aptos-move/framework/move-stdlib/sources/bcs.move

```diff
@@ -13,5 +13,14 @@ module std::bcs {
     spec module {
         /// Native function which is defined in the prover's prelude.
         native fun serialize<MoveValue>(v: &MoveValue): vector<u8>;
+
+        fun spec_to_bytes<MoveValue>(v: MoveValue): vector<u8>;
```

Comment:
You don't need to define this spec function. Instead, you can use `serialize` for the same purpose.